packages/zoe/src/contractSupport/bondingCurves.js

```diff
@@ -1,7 +1,10 @@
 import { assert, details } from '@agoric/assert';
+import Nat from '@agoric/nat';
 import { natSafeMath } from './safeMath';
 
-const { add, subtract, multiply, floorDivide } = natSafeMath;
+const { multiply, floorDivide } = natSafeMath;
+const BIG_10000 = BigInt(10000);
```

Comment:
Just add a TODO comment explaining that we say `BigInt(1000)` rather than `1000n` because some tool somewhere in our toolchain is using some JavaScript parser that does't know how to parse bigint literal syntax. (Even better if you can say what that tool is.)

Reply:
Even if `10000n` were legal/worked, I'd still prefer the manifest constant because typos like your `1000n` would be caught more easily.   Maybe BIG_TEN_THOUSAND would be even clearer.

Reply:
Even then we would rewrite this line as
```js
const BIG_SOMETHING = 1000n;
```

Reply:
Got it.  Still a shared manifest constant, but preferentially using the `n` notation.